ush/run_mpmd.sh

```diff
@@ -41,6 +41,7 @@ elif [[ "${launcher:-}" =~ ^mpiexec.* ]]; then  # mpiexec
     ((nm=nm+1))
   done < "${cmdfile}"
 
+  launcher=${APRUN_DWN}
```

Comment:
run_mpmd.sh is generic. 
APRUN_DWN is specific. 
Thanks for catching this, I am not sure how this escaped on WCOSS2 testing of CFP. 
We will need this to be corrected, but we can do that in another PR after this one is merged.

Reply:
@aerorahul This line is my workaround. 
APRUN_DWN='mpiexec -l -np 24 --cpu-bind verbose,core cfp'
With this setting, the MPMD runtime is within 1.5 minutes.
I saw APRUN_DWN setting from WCOSS2.env in the G-W version for HR2, but not in current develop branch. BTW, I will remove my change from this PR. We can work this later.

Reply:
Ok. Understood.
Would you be willing to test the patch in https://github.com/NOAA-EMC/global-workflow/pull/1973#issuecomment-1779290357 in lieu of your workaround?
I am ok with including it in this PR or promoting this to a hotfix and getting it in sooner.